Getting_Started.md

```diff
@@ -0,0 +1,128 @@
+# Getting Started with Bunsen
+
+This guide is intended to help you get your first Bunsen environment up and running.
+
+# Expectations
+
+This project is simply an ansible playbook that has a certain set of requirements that need to be satisfied moving in.
```

Comment:
I'm getting stuck on 'moving in'. Maybe change to 'before proceeding'?

Reply:
before moving on i think is what he wants there

Reply:
The suggestion below from @jshimkus-rh is closer to what I had in my mind when writing this.

I will provide my response in both threads for consideration.

```suggestion
This project is an ansible playbook with specific requirements that need to be satisfied before it can be used to provision an environment.
```

Reply:
This is addressed in my recent pushes.